.travis.yml

```diff
@@ -77,10 +80,14 @@ matrix:
            CONDA_DEPENDENCIES=${CONDA_ALL_DEPENDENCIES}
            EVENT_TYPE='cron'
 
+
 install:
+  - sudo apt-get install libssl1.0.0
   - git clone git://github.com/astropy/ci-helpers.git
```

Comment:
Clone git://github.com/astropy/ci-helpers.git (in fact, maybe MDAnalysis should have a clone... – @kain88-de ?) and then pull from the clone with your changes.

Contribute your changes (once working) as PR to upstream.